renderdoc/driver/shaders/spirv/CMakeLists.txt

```diff
@@ -76,6 +76,11 @@ set(sources
     spirv_disassemble.cpp
     ${glslang_sources})
 
+if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+    set_source_files_properties(${glslang_dir}/SPIRV/GlslangToSpv.cpp
+        PROPERTIES COMPILE_FLAGS "-Wno-tautological-constant-out-of-range-compare")
```

Comment:
I've generally tried to make a PR upstream for any warning fixes to glslang, but this is fine for now.